recipes/xbitinfo/install_julia.patch

```diff
@@ -0,0 +1,204 @@
+diff --color -Naur xbitinfo-0.0.2.orig/setup.py xbitinfo-0.0.2/setup.py
+--- xbitinfo-0.0.2.orig/setup.py	2022-07-11 17:37:19.000000000 -0300
++++ xbitinfo-0.0.2/setup.py	2022-08-29 10:08:18.745686078 -0300
+@@ -5,26 +5,6 @@
+ import os
+ 
+ from setuptools import find_packages, setup
+-from setuptools.command.develop import develop
+-from setuptools.command.install import install
+-
+-julia_install_command = "julia install_julia_packages.jl"
+-
+-
+-class PostDevelopCommand(develop):
+-    """Post-installation for development mode."""
+-
+-    def run(self):
+-        develop.run(self)
+-        os.system(julia_install_command)
+-
+-
+-class PostInstallCommand(install):
+-    """Post-installation for installation mode."""
+-
+-    def run(self):
+-        install.run(self)
+-        os.system(julia_install_command)
+ 
+ 
+ with open("README.md") as readme_file:
+@@ -66,10 +46,6 @@
+         "Programming Language :: Python :: 3.9",
+         "Programming Language :: Python :: 3.10",
+     ],
+-    cmdclass={
+-        "develop": PostDevelopCommand,
+-        "install": PostInstallCommand,
+-    },
+     description="Retrieve information content and compress accordingly.",
+     install_requires=requirements,
+     license="MIT license",
+diff --color -Naur xbitinfo-0.0.2.orig/xbitinfo/__init__.py xbitinfo-0.0.2/xbitinfo/__init__.py
+--- xbitinfo-0.0.2.orig/xbitinfo/__init__.py	2022-07-11 17:37:19.000000000 -0300
++++ xbitinfo-0.0.2/xbitinfo/__init__.py	2022-08-29 10:49:44.734576952 -0300
+@@ -1,7 +1,8 @@
+ """Top-level package for xbitinfo."""
+ 
+ from ._version import __version__
+-from .bitround import jl_bitround, xr_bitround
+-from .graphics import plot_bitinformation, plot_distribution
+-from .save_compressed import get_compress_encoding_nc, get_compress_encoding_zarr
+-from .xbitinfo import get_bitinformation, get_keepbits, get_prefect_flow
++from .julia_helpers import install
++# from .bitround import jl_bitround, xr_bitround
++# from .graphics import plot_bitinformation, plot_distribution
++# from .save_compressed import get_compress_encoding_nc, get_compress_encoding_zarr
++# from .xbitinfo import get_bitinformation, get_keepbits, get_prefect_flow
+diff --color -Naur xbitinfo-0.0.2.orig/xbitinfo/julia_helpers.py xbitinfo-0.0.2/xbitinfo/julia_helpers.py
+--- xbitinfo-0.0.2.orig/xbitinfo/julia_helpers.py	1969-12-31 21:00:00.000000000 -0300
++++ xbitinfo-0.0.2/xbitinfo/julia_helpers.py	2022-08-29 14:03:23.819518599 -0300
+@@ -0,0 +1,128 @@
++"""Functions for initializing the Julia environment and installing deps."""
++import warnings
++from pathlib import Path
++import os
++
++from ._version import __version__
++
++
++def install(julia_project=None, quiet=False):  # pragma: no cover
```

Comment:
@observingClouds the changes here are a copy from the pysr approach to install the juilia dependencies.